packages/integration-sdk-cli/src/__tests__/util/recording.ts

```diff
@@ -0,0 +1,16 @@
+import { Polly, PollyConfig } from '@pollyjs/core';
+
+export function createTestPolly(
+  recordingName: string,
+  pollyConfigOverrides?: PollyConfig,
+) {
+  return new Polly(recordingName, {
+    adapters: ['node-http'],
+    persister: 'fs',
+    logLevel: 'silent',
```

Comment:
The only change made when creating new `Polly` instances is here. There used to be a `logging` boolean, which was assigned `false`. In Polly.JS v6, `logging` was removed and replaced with `logLevel`. I kept the existing behavior by assigning `silent` as the value.